chain/pool/src/lib.rs

```diff
@@ -253,11 +271,11 @@ mod tests {
         mut transactions: Vec<SignedTransaction>,
         expected_weight: u32,
     ) -> (Vec<u64>, TransactionPool) {
-        let mut pool = TransactionPool::new(TEST_SEED);
+        let mut pool = TransactionPool::new(TEST_SEED, TRANSACTION_POOL_SIZE);
         let mut rng = thread_rng();
         transactions.shuffle(&mut rng);
         for tx in transactions {
-            pool.insert_transaction(tx);
+            assert!(pool.insert_transaction(tx).is_ok());
```

Comment:
It’s probably better to `unwrap`. While at the moment the method can fail for only one reason, the benefit of `unwrap` (apart from being shorter to write) is that it will output the error value if the result is not OK.

Applies below as well.